docs/getting-started.md

```diff
@@ -9,19 +9,37 @@ Before proceeding, install the following prerequisites:
 Once everything is installed, execute the following commands from project root:
 
 ```bash
-cd docker/
 docker-compose up -f docker/docker-compose.yml -d
 ```
 
 ## User management
 
 ### Account creation
 
-TBD
+Use the Mainflux manager HTTP/REST API to create user account:
```

Comment:
`Use the Mainflux API to create user account:`. Drop REST/HTTP wherever not necessary.